canonical-web-and-design/tutorials.ubuntu.com
# [iot] update kiosk guide to reflect snapd/snapcraft changes

## Done

Update the Mir-kiosk IoT guides to reflect:
- adopt core18 by default
- use newer snapcraft/snapd features

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8016/](http://0.0.0.0:8016/)
- Follow the tutorials and ensure the steps are correct
